tls/src/main/java/org/bouncycastle/tls/NamedGroup.java

```diff
@@ -102,6 +102,10 @@ public class NamedGroup
     public static final int arbitrary_explicit_prime_curves = 0xFF01;
     public static final int arbitrary_explicit_char2_curves = 0xFF02;
 
+    public static final int kyber512 = 0x023A;
```

Comment:
@hwupathum Note that that issue is stale. In 0.10.0, liboqs now includes draft ML-KEM in addition to round 3 Kyber.

Reply:
Yes, but as I mentioned the algorithm IDs are different between liboqs and BC.

Reply:
@hwupathum Yes, I would expect the first to fail, because Round 3 Kyber and draft ML-DSA are different.

The latter I would've expected to succeed. I'd suggest we should fix the compatibility for this case.

I don't think support for Round 3 Kyber will remain once NIST publishes drafts. I'd imagine both Cloudflare and Google will update fairly quickly to the standardized (non-draft) ML-KEM implementation.